builtin/providers/azurerm/resource_arm_availability_set.go

```diff
@@ -93,6 +102,13 @@ func resourceArmAvailabilitySetCreate(d *schema.ResourceData, meta interface{})
 		Tags: expandTags(tags),
 	}
 
+	if managed == true {
+		n := "Aligned"
```

Comment:
I should also not that there did not appear to be a constant for that value defined in the SDK either.

https://github.com/Azure/azure-sdk-for-go/search?q=Aligned&type=Code&utf8=%E2%9C%93